eduroam/nmcli.md

```diff
@@ -28,22 +28,22 @@ All commands starting with `$` can be used as your standard user. All commands u
 
 2. Grab the name of the wireless network interface  that you want to use for this connection. You will need this throughout this setup process. You can list all network interfaces by running the command:
 ```
-$ nmcli
+$ nmcli device show
 ```
+>Check the one whose `GENERAL.TYPE` is `wifi`. Most commonly, this is `wlan0` or `wlp5s0`.
 3. Open a terminal and use nmcli to create a new wifi connection on your wireless network interface that connects to the `eduroam` network:
 ```
-$ nmcli connection add type wifi ifname <your interface> con-name <connection name> ssid eduroam
+$ nmcli connection add type wifi ifname <your interface> con-name eduroamWiFi ssid eduroam
```

Comment:
Is there a particular reason this was done? i made it into a placeholder to hopefully make it easier to parse through the command visually to see what was a placeholder that needed replacing in case someone wanted to name it something else.

Reply:
I made it this way since the reader already has one variable kept track of in their head `<your-interface>`. 

This is debatable though. 
If you disagree with this change, I implore you to edit the step where you "open the file in your editor of choice".  The step reads "Open the `eduroam.nmconnection` file in the editor of your choice *as root*". However, this file name would actually be "<con-name>.nmconnection".